src/object.cpp

```diff
@@ -544,6 +544,17 @@ Object *Object::GetChild(int idx) const
     return m_children.at(idx);
 }
 
+Object *Object::GetChild(int idx, const ClassId classId)
+{
+    ArrayOfObjects objects;
+    ClassIdComparison matchClassId(classId);
+    this->FindAllChildByComparison(&objects, &matchClassId);
```

Comment:
I think we want to add a deepness parameter of 1, don't we?

Reply:
Thanks! 797169a6f876eaf772e885e3c5d03b0e18d4d5fc